src/SFA.DAS.EmployerAccounts.Web/Views/EmployerAccountPaye/WaysToAdd.cshtml

```diff
@@ -39,6 +39,16 @@
                         I want to use my account office reference number and employer PAYE scheme reference
                     </label>
                 </fieldset>
+                <p>
+                    <div class="notice">

```

Comment:
Should we not be suing [gds code](https://design-system.service.gov.uk/components/warning-text/#warning-text-second-example-html) for this?

Reply:
That's the wrong GDS for our project.

Reply:
My mistake, apologies.